src/test/java/com/mongodb/kafka/connect/util/SimpleDateTimeParserTest.java

```diff
@@ -0,0 +1,44 @@
+/*
+ * Copyright 2008-present MongoDB, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.mongodb.kafka.connect.util;
+
+import static org.junit.jupiter.api.Assertions.assertAll;
+import static org.junit.jupiter.api.Assertions.assertEquals;
+
+import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.api.Test;
+
+public class SimpleDateTimeParserTest {
+
+  @Test
+  @DisplayName("test default date string")
+  void testDefaultDateStrings() {
+    SimpleDateTimeParser simpleDateTimeParser =
+        new SimpleDateTimeParser("yyyy-MM-dd['T'][ ]HH:mm:ss[.][SSSSSS][SSS][ ][X]");
```

Comment:
The pattern here and in `MongoSinkConfigTest` are specified explicitly instead of being specified via the constant `TIMESERIES_TIMEFIELD_AUTO_CONVERSION_DATE_FORMAT_DEFAULT`. If we change the default pattern in the future (if the new pattern broadens the spectrum of strings that fit it, it is a backward-compatible change), neither of the tests will automatically pick up the new default value.

If marking `MongoSinkTopicConfig.TIMESERIES_TIMEFIELD_AUTO_CONVERSION_DATE_FORMAT_DEFAULT` as `public` is not a good idea, maybe we could move the constant somewhere where it can be public and then reuse in other places, including `MongoSinkTopicConfig`? Or maybe it can be just made public in `MongoSinkTopicConfig`.

Reply:
Done.